README.md

```
@@ -86,8 +92,8 @@ Output to /build.
 dart run rps build-release web
 ```
 
-### Using Docker Compose
+## Using Docker Compose
 You can start the Web server in a Docker environment using the following command:
 ```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)